utils/python/tests/scripts_regression_tests.py

```diff
@@ -463,6 +463,7 @@ def tearDown(self):
         else:
             for file_to_clean in files_to_clean:
                 if (os.path.isdir(file_to_clean)):
+                    time.sleep(5) # Kinda hacky
```

Comment:
I don't think I can accept this, it will slow down testing dramatically. What are we trying to solve here? Can we limit this to certain machines? Can you put the sleep outside of the loop?